content/docs/user-guide/visualization/environments/terrain/terrain-developer-guide/future-work.md

```diff
@@ -0,0 +1,55 @@
+---
+linktitle: Future Work
+title: Future Work
+description: Possible future improvements to the terrain system.
+weight: 500
+toc: true
+---
+
+There are a number of improvements that could potentially be made to the terrain system for anyone interested in taking the system further.
+
+## Large-scale improvements
+
+This category of improvements requires architecting new solutions to provide extensive new capabilities.
+
+### 3D geometry support
+
+Instead of a heightfield-based approach, the APIs and renderer could be extended to use a full 3D approach, whether it's through voxels, displacement maps, mesh ingestion, raytracing, or some other technology. This would enable the system to support caves, arches, cliff geometry, and other vertical or overlapping terrain features that simply can't be supported through a pure heightfield approach.
+
+The API extensions would likely consist of redefining the current APIs to provide the _highest elevation_ terrain data that appears below the given query input positions, and then adding more APIs to provide _all_ the terrain data that exists at or below the given query positions.
```

Comment:
Are you sure? I think "may" means "possible", but doesn't imply "likely". (The definition of "may" is _literally_ just "expressing possibility.")

If the goal is to remove "would", how about "will likely" instead of "would likely"? This is one place that I can understand the would/will swap, since the addition of "likely" keeps it in the hypothetical instead of the absolute.

Reply:
Yeah, at least compared to "might", "may" indicates greater chance of possibility. I understand that probably isn't intuitive for readers though. "will likely" is perfectly fine